src/index.js

```diff
@@ -2,6 +2,7 @@ import Animations from './props.animations.js'
 import Sizes from './props.sizes.js'
 import Colors from './props.colors.js'
 import ColorsHSL from './props.colors-hsl.js'
+import Container from "./props.container.js";
```

Comment:
```suggestion
import Container from './props.container.js'
```
match style of others